hal/halx86/apic/apictimer.c

```diff
@@ -63,22 +69,75 @@ VOID
 NTAPI
 HalStartProfileInterrupt(IN KPROFILE_SOURCE ProfileSource)
 {
-    UNIMPLEMENTED;
-    return;
+	LVT_REGISTER LvtEntry;
```

Comment:
Style notes: could you use 4 space indentation instead of tabs please, and always put braces around if/else blocks? Thanks.

Reply:
Okay. I'm usually don't use IDEs such as MSVS, so it just took me very long to figure out how to make it insert spaces instead of tabs. Will fix this.